### PR TITLE
`SuspenseProps` extension is moved to separate file

### PR DESCRIPTION
**Pull request by sgrishchenko**
cc @turansky

### CODE REVIEW

File: kotlin-react/src/main/kotlin/react/Suspense.ext.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import react.Render
+import react.SuspenseProps
+import react.createElement
+
+// 16.6+
+fun SuspenseProps.fallback(handler: Render) {
+    asDynamic().fallback = createElement(handler)
```
Comment (turansky) [inline kotlin-react/src/main/kotlin/react/Suspense.ext.kt:7]: Redundant `asDynamic`?

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-react/src/main/kotlin/react/Suspense.ext.kt (hunk 1 above)
Comment (sgrishchenko) [reply kotlin-react/src/main/kotlin/react/Suspense.ext.kt:7]: `SuspenseProps` are decalared next way now:
```
external interface SuspenseProps : PropsWithChildren
```
Shell I declare it this way:
```
external interface SuspenseProps : PropsWithChildren {
    var fallback: ReactNode?
}
```

Comment (turansky) [reply kotlin-react/src/main/kotlin/react/Suspense.ext.kt]: [Yes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b8e894eaa4ce70830a40a6bcc3c12392eb867989/types/react/index.d.ts#L400)

Comment (sgrishchenko) [reply kotlin-react/src/main/kotlin/react/Suspense.ext.kt]: fixed

**Review by turansky — APPROVED** [review]

**Pull request merged**